src/Table/components/TableContent.js

```diff
@@ -19,28 +20,33 @@ export const TableContent = ({ titleBarVisible, dataHook }) => {
           hideHeader: !titleBarVisible,
         };
 
-        if (tableProps.showSelection) {
-          return (
-            <BulkSelectionConsumer
-              consumerCompName="Table.Content"
-              providerCompName="Table"
-            >
-              {bulkSelectionContext => (
-                <DataTable
-                  {...dataTableProps}
-                  isRowSelected={(rowData, rowIndex) =>
-                    bulkSelectionContext.isSelected(
-                      defaultTo(rowData.id, rowIndex),
-                    )
-                  }
-                  columns={createColumns({ tableProps, bulkSelectionContext })}
-                />
-              )}
-            </BulkSelectionConsumer>
-          );
-        } else {
-          return <DataTable {...dataTableProps} />;
-        }
+        return (
+          <div {...styles('content', { titleBarVisible })}>
```

Comment:
```suggestion
          <div {...styles('content', { titleBarVisible }, this.props)}>
```

this helps to avoid the notorious `red block of error` and also allows users to extend styles in `.st.css` files.

if i remember correctly, there's a promise that it's no longer required in stylable v2, but we're not using that yet

Reply:
added props - slightly different since this is a functional component